tests/unit/swagger/definitionsGeneration/metadata.spec.ts

```diff
@@ -419,7 +419,7 @@ describe('Metadata generation', () => {
       expect(genderParam.parameterName).to.equal('gender');
       expect(genderParam.description).to.equal('Gender description');
       expect(genderParam.required).to.be.true;
-      expect(genderParam.type.dataType).to.equal('enum');
```

Comment:
This should not change this, should it?

Reply:
Issue did not appear to be fixed on 3.x, and the change is so simple that I kept it in.